lib/presenter.rb

```diff
@@ -0,0 +1,19 @@
+class Presenter
+  def initialize
+    @board = "
+             |       |
+         1   |   2   |   3
+       _____ | _____ | _____
+             |       |
+         4   |   5   |   6
+       _____ | _____ | _____
+             |       |
+         7   |   8   |   9
+             |       |
+        "
+  end
+
```

Comment:
You don't really need an initialize method for the presenter.  You could just have the board in print_board.

Reply:
Ok, so the Presenter just has a print_board function?